DcCore/DcCore/Helper/DcColors.swift

```diff
@@ -1,6 +1,9 @@
 import UIKit
 
 public struct DcColors {
+    private static let white						= #colorLiteral(red: 1, green: 1, blue: 1, alpha: 1)
```

Comment:
this is also a tab, no spaces, no big deal here, however @nayooti please fix your environment :)